services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java

```diff
@@ -108,6 +109,10 @@ public static Response getResponse(KeycloakSession session, Throwable throwable)
                     .build();
         }
 
+        if (throwable instanceof SendResponseException sre) {
```

Comment:
Would it work if `SendResponseException` is a `WebApplicationException` and we change the error handler to always check whether the thrown exception is of the type of the latter and has a response, and if so, assume the response from it should be returned?

I'm wondering if we can make the handler more flexible while still aligned with Jakarta RS API.